README.md

```diff
@@ -1,6 +1,6 @@
 # PM2 monitoring tool for Zabbix
 
-## What it is
+## What is it?
```

Comment:
should be burned to the ground!

Reply:
yes, that's exactly what is wrong with #Hacktoberfest 😢

Reply:
I think that's incompatible with this markdownlint rule: https://github.com/updownpress/markdown-lint/blob/master/rules/026-no-trailing-punctuation.md